vllm/engine/llm_engine.py

```diff
@@ -974,3 +990,48 @@ def list_loras(self) -> List[int]:
 
     def check_health(self) -> None:
         self.model_executor.check_health()
+
+    def create_trace_span(self, seq_group: SequenceGroup, now: float) -> None:
+        if seq_group.trace_context is None:
+            return
+        arrival_time_nano_seconds = int(seq_group.metrics.arrival_time * 1e9)
+
+        with tracer.start_as_current_span(
```

Comment:
QQ: what's the e2e overhead of this API? Can you measure and lmk for prompt length 512 output length 128?